.make/test.mk

```diff
@@ -577,8 +577,10 @@ clean-coverage-remote:
 
 CLEAN_TARGETS += clean-e2e
 .PHONY: clean-e2e
-## Removes the end-to-end (e2e) test directory
```

Comment:
Why did you remove the description for this clean target? It will be shown when running `make help` which is quite nice.